waku/waku_store/common.nim

```diff
@@ -27,13 +27,15 @@ type WakuStoreResult*[T] = Result[T, string]
 
 type MessageDigest* = MDigest[256]
 
-proc computeDigest*(msg: WakuMessage): MessageDigest =
+proc computeDigest*(msg: WakuMessage, pubSubTopic: string): MessageDigest =
```

Comment:
What if we completely remove this and only have the `computeDigest` from `waku_archive`?
I suggest doing that in a separate PR.

Reply:
indeed, this created a whole lot of mess for me earlier.